### ISSUE TITLE
Cannot reinitialize an instance on the same textbox

### ISSUE THREAD

**Issue by tno2007** [immense/immybox#20]
If i run these two scripts on the same textbox, then only the first one's options shows:

    $('#txtOption').immybox({
        choices: [
            { text: 'Monday', value: '1' },
            { text: 'Tuesday', value: '2' }
        ]
    });

    $('#txtOption').immybox({
        choices: [
            { text: 'January', value: '1' },
            { text: 'February', value: '2' }
        ]
    });

In other words, "Monday, Tuesday..." will be shown, instead of "January, February..."

**Comment by cbankester** [comment]
You can use the `setChoices` method, e.g.:
```javascript
$('#txtOption').immybox({choices: [{text: 'Foo', value: 'foo'}, {text: 'Bar', value: 'bar'}]});
$('#txtOption').immybox('setChoices', [{text: 'Foobar', value: 'foobar'}, {text: 'Barbaz', value: 'barbaz'}]);
```
